src/TR_alg.jl

```diff
@@ -179,7 +177,7 @@ function TR(
       continue
     end
 
-    subsolver_options.ϵa = k == 1 ? 1.0e-5 : max(ϵ_subsolver, min(1e-2, sqrt_ξ1_νInv))
+    subsolver_options.ϵa = k == 1 ? 1.0e-5 : max(ϵ_subsolver, min(1e-2, sqrt_ξ1_νInv) * ξ1)
```

Comment:
@geoffroyleconte I reinstated this because it disappeared in [ac255073a980ee340c02d7dff568dc0f2a07d265](ac255073a980ee340c02d7dff568dc0f2a07d265). It's not clear which produces the best results.

Reply:
It was intended, and this choice is mentioned in the section numerical results of indef-pg.